src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -22,8 +22,18 @@
                            'Exception will be thrown if free instances exceed one.')
 register_cli_argument('iot hub create', 'unit', help='Units in your IoT Hub.', type=int)
 
+# Arguments for 'iot hub show-connection-string'
+register_cli_argument('iot hub show-connection-string', 'name', options_list=('--name', '-n'),
+                      help='Target IoT Hub name.')
```

Comment:
>) [](start = 49, length = 1)

I would recommend you add a completer here to enable tab completion of hub names.